target/linux/realtek/patches-6.12/723-net-mdio-Add-Realtek-Otto-auxiliary-controller.patch

```diff
@@ -209,6 +217,10 @@ Signed-off-by: Sander Vanheule <sander@svanheule.net>
 +		.compatible = "realtek,rtl9300-aux-mdio",
 +		.data = &info_rtl930x,
 +	},
++	{
++		.compatible = "realtek,rtl9310-aux-mdio",
++		.data = &info_rtl931x,
++	},	
```

Comment:
Nitpick: please remove the extra tab at the end of the line

Reply:
fixed